### PR TITLE
Extend known bugs for Cloud TW image

### PR DESCRIPTION
**Pull request by mloviska**
http://kepler.suse.cz/tests/23916#step/journal_check/8

### CODE REVIEW

File: data/journal_check/bug_refs.json [hunk 1]
```diff
@@ -692,5 +693,26 @@
             "opensuse": ["Tumbleweed"]
         },
         "type": "bug"
+    },
+    "boo#1227900": {
+        "description": "Speculative Return Stack Overflow: IBPB-extending microcode not applied!|Speculative Return Stack Overflow: WARNING: See https://kernel.org/doc/html/latest/admin-guide/hw-vuln/srso.html for mitigation options.",
+        "products": {
+            "opensuse": ["Tumbleweed"]
+        },
+        "type": "bug"
```
Comment (Vogtinator) [inline data/journal_check/bug_refs.json:702]: Host issue, not SUT issue.

```suggestion
        "type": "ignore"
```

Comment (mloviska) [reply data/journal_check/bug_refs.json]: fixed

File: data/journal_check/bug_refs.json [hunk 2]
```diff
@@ -692,5 +693,26 @@
...
+    },
+    "boo#1227908": {
+        "description": "Option builtin_af_unix line \\d+ is obsolete - using /sbin/audisp-af_unix|Option builtin line \\d+ is obsolete - update it",
+        "products": {
+            "opensuse": ["Tumbleweed"]
+        },
+        "type": "bug"
+    },
+    "amd_pstate": {
```
[the remark this answers is not in the document]
Comment (mloviska) [reply data/journal_check/bug_refs.json]: reported, thanks

[the remark this answers is not in the document]
Comment (mloviska) [reply data/journal_check/bug_refs.json]: updated according to bzilla latest comment

**Review by Vogtinator — APPROVED** [review]

**Pull request merged**